.github/workflows/test.yml

```
@@ -85,6 +85,8 @@ jobs:
           java-version: '11'
           cache: 'sbt'
       - name: Integration Tests
+        if: github.event_name == 'pull_request' &&
+          ! contains(github.event.pull_request.labels.*.name, 'Skip chiseltest')
```

```suggestion
        if: github.event_name != 'pull_request' ||
          ! contains(github.event.pull_request.labels.*.name, 'Skip chiseltest')
```
The guard against `pull_request` is needed to check the labels, but we should still run these tests for other event types (`push`).